Classes/Service/Typo3Service.php

```diff
@@ -55,11 +55,11 @@ class Typo3Service implements SingletonInterface
      */
     public function __construct(Sequenzer $sequenzer)
     {
-        $this->conf = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['aoe_dbsequenzer']);
+        $this->conf = $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['aoe_dbsequenzer'];
```

Comment:
We should use the `GeneralUtility::makeInstance(ExtensionConfiguration::class)->get('aoe_dbsequenzer')` instead, then we would have avoided a problem like this with "hardcode" path to the setting, and the serialized.

Reply:
Changed it like recommanded and additional remove the deprecated devLog stuff